lib/linguist/languages.yml

```diff
@@ -4016,13 +4018,13 @@ YAML:
   aliases:
   - yml
   extensions:
-  - .yml
   - .reek
   - .rviz
   - .sublime-syntax
   - .syntax
   - .yaml
   - .yaml-tmlanguage
+  - .yml
```

Comment:
This needs to go back to the top of the list of extensions. The first in the list is considered the 'primary' extension for the language.